x-pack/plugins/data_enhanced/server/search/session/session_service.ts

```diff
@@ -360,7 +347,7 @@ export class SearchSessionService implements ISessionService {
       const attributes = {
         idMapping: { [requestHash]: searchInfo },
       };
-      await this.update(sessionId, attributes, deps);
+      await this.update(deps, sessionId, attributes);
```

Comment:
We talked about letting the monitoring loop pick up new IDs.
Was there any good reason for this to be changed? I think this will lead to more 409's and possible race conditions.